src/components/org-users/org-users.ts

```diff
@@ -388,13 +388,15 @@ export async function inviteUser(ctx: IContext, params: IParameters, body: objec
       logger: ctx.app.logger,
     });
 
-    const uaaUser = await uaa.findUser(values.email);
+    const emailAddress = values.email.toLowerCase();
+
+    const uaaUser = await uaa.findUser(emailAddress);
```

Comment:
Just to confirm to myself that I wasn't misunderstanding, I checked that `findUser` already ignores case:

> Attribute operators 'eq', 'co', and 'sw' MUST perform caseIgnore matching for all string attributes
http://www.simplecloud.info/specs/draft-scim-api-01.html#query-resources

So this PR just changes the invitation address and the one in paas-accounts to be lowercase, which seems fine.